packages/dashboard/src/widgets/Widget.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {
+  CircularProgress,
+  Paper,
+  Box,
+  LocaleKey,
+  Typography,
+  useTranslation,
+} from '@openmsupply-client/common';
+
+const Loading = () => (
+  <Box display="flex" flex={1} justifyContent="center" alignItems="center">
+    <CircularProgress />
+  </Box>
+);
+
+interface WidgetProps {
+  height?: number | string;
+  titleKey: LocaleKey;
+}
+
+const Widget: React.FC<WidgetProps> = ({
+  children,
+  height = '100%',
+  titleKey,
+}) => {
+  const t = useTranslation();
+  return (
+    <Paper
+      style={{
+        borderRadius: 16,
+        height,
+        padding: 24,
+        width: 400,
+        display: 'flex',
+        flexDirection: 'column',
+        margin: 11,
+      }}
+    >
+      <Typography sx={{ fontSize: '14px', fontWeight: 'bold' }}>
+        {t(titleKey)}
+      </Typography>
+
+      <React.Suspense fallback={<Loading />}>{children}</React.Suspense>
+    </Paper>
```

Comment:
This is cool. I can easily see the dashboard remote loading plugin components to display which would be pretty wicked

Reply:
would be nice to move the widgets into their functional areas ..and load them up suspensfully 😄